net/dcwifi/macremapper/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2019 EWSI
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
+
+PKG_NAME:=macremapper
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ewsi/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=f054201dd805ce005b89606a507b58a5717d383a4339c69dfdc02f0202935437
+
+PKG_MAINTAINER:=Carey Sonsino <careys@edgewaterwireless.com>
+PKG_LICENSE:=Apache-2.0
```

Comment:
This should be GPL-2.0-only

Reply:
I was planning on using `GPL-2.0-or-later` in the file headers (see https://github.com/ewsi/macremapper/pull/3).  Is that incorrect and I should use `GPL-2.0-only` instead?

Reply:
I set `PKG_LICENSE:=GPL-2.0-or-later` to be consistent with that other PR

Reply:
Kernel modules are typically GPL-2.0-only as that is what the kernel itself is licensed as. Of course you can change the license of the module as you wish. I just saw the COPYING file and assumed GPL-2.0-only.

Reply:
I ended up switching everything to `GPL-2.0-only`...